javascript/javascript1/week1/index.js

```diff
@@ -3,29 +3,26 @@
 const yearOfBirth = 1983;
 const yearFuture = 2023;
 const age = yearFuture - yearOfBirth;
-const result = (`You will be ${age} years old in ${yearFuture}.`);
+const result = `You will be ${age} years old in ${yearFuture}.`;
 /* or const result = ("You will be " + age + " years old in" + yearFuture + ".") */
 
 console.log(result);
 
 /* =======================================================================================*/
 
-
 /* HYF HOMEWORK - TASK 2 - Goodboy-Oldboy (A dog age calculator) */
 
 const dogYearOfBirth = 2000;
 const dogYearFuture = 2010;
 const dogYear = (dogYearFuture - dogYearOfBirth) * 7;
 const shouldShowResultInDogYears = false;
 
```

Comment:
Same here. You could mention the variable data type as asked :)

Reply:
Look @ the previous comment ;)  All set 'n done !